langs/python.go

```diff
@@ -118,8 +118,8 @@ def handler(ctx, data: io.BytesIO=None):
 
 func (h *PythonLangHelper) DockerfileCopyCmds() []string {
 	return []string{
-		"COPY --from=build-stage /function /function",
 		"COPY --from=build-stage /python /python",
-		"ENV PYTHONPATH=/python",
+		"COPY --from=build-stage /function /function",
+		"ENV PYTHONPATH=/function:/python",
```

Comment:
I feel like this is not the place to do the following change, FDK is where I believe this must be added to make sure that Python FDK CLI will be capable to do modules discovery starting from the root folder (that would be `/function`

Reply:
I'm not sure I follow this; the change works as-is. I can't find any use of `sys.path` ni the FDK itself - it looks to rely on this setting in order to operate.

Reply:
I mean users must be aware that if they go with built-in dockerfile then the `PYTHONPATH` would contain not just the interpreter path but `/function` folder as well.

In order to make this process versatile and `PYTHONPATH`-independent, Python FDK must be capable to process the root folder of a function but in a manner of the delayed imports. Meaning, the root folder modules and packages must be initialized only when user's code gets executed or it will turn into a nightmare of imports profiling.

Reply:
This seems to behave okay under testing.

What I've seen from actual customer use is typically the addition of small helper modules - nothing large, but nonetheless inconvenient to pull into a single `func.py`.

Given that there's a need for this now - is it possible to act, then iterate?